spec/service-discovery.md

```diff
@@ -354,56 +352,33 @@ TLS channel created.
-federated service given the value to its service mesh ingress, the owner is free
-to choose the format it prefers.
+For each `FedeatedService`, the consumer agent must create in the consumer platform an entry in the local service registry (for example, in the local DNS) with an FQDN for each of the `Instance` with the format `FederatedServiceInstance.spec.instanceID+FederatedService.spec.fqdn`. In addition, the consumer agent must create in the local service registry an entry with `FederatedService.spec.fqdn` and all the information in each `Instance` and corresponding `Endpoint`.
 
 **Request message format**
 
 | Parameter | Data type | Required | Description |
 | --- | --- | --- | --- |
-| `Name` | `string` | No | Human readable name for the service |
-| `FQDN` | `string` | Yes | FQDN the federated service exposes outside of the owner mesh |
-| `ServiceID` | `string` | Yes | Name assigned to the service in a particular owner SM. It must be unique in the owner SM and will be the value sent as the SNI header by the consumer SM to a particular owner SM. Each SM vendor will possibly have a different kind of SNI and thus this spec does not define a specific format. |
-| `SAN` | `string` | Yes | URI SAN of the Federated Service to enable end to end security, in SPIFFE format |
-| `MeshIngress` | `Array<Endpoint>` | Yes | The endpoints of the ingress of the mesh where the service endpoints can be reached |
-| `Protocols` | `Array<string>` | Yes | Protocols associated with the service endpoint |
-| `Description` | `string` | No | Description of the service |
-| `Tags` | `Array<string>` | No | Informative values for filtering purposes |
-| `Labels` | `Map<string, string>` | No | Informative array of KV for filtering purposes |
+| `Name` | `string` | Yes | The unique name that identifies the service within an owner platform, and serves as its unique ID. Uniqueness in the service owner platform. The consumer platform must take into account possible name collistions that may happen with other FederatedService consumed from other owner platforms through, for example, aliasing.|
+| `FQDN` | `string` | Yes | The FQDN (Fully Qualified Domain Name) the consumer platform uses as a local DNS name to access the imported FederatedService. The consumer platform DNS is programmed to reference the Endpoint of the imported FederatedService. It is unique within the service owner platform. The consumer platform must solve any possible naming collistions that may happen with other FederatedService consumed from other owner platforms through, for example, aliasing. |
+| `SANS` | `Array<string>` | No | List of URI SANs A list of SANs (Subject Alternative Names) to enable end-to-end security for the federated service. |
+| `Instances` | `Array<Instance>` | Yes | Represents an instance of the FederatedService. |
+| `Endpoints` | `Array<Endpoint>` | Yes | The endpoints that provide access to a FederatedService in the owner platform. For example, this can be an ingress. |
```

Comment:
```suggestion
| `Endpoints` | `Array<Endpoint>` | Yes | The endpoints that provide access to a `FederatedService` in the owner platform. For example, this can be an ingress. |
```